src/menus.ts

```diff
@@ -1,14 +1,57 @@
-import { distanceInWords } from 'date-fns';
+import { distanceInWords, isWithinRange, subMonths } from 'date-fns';
 import { MenuItemConstructorOptions, shell } from 'electron'; // tslint:disable-line no-implicit-dependencies
+import IncidentFeed from './incidentFeed';
 import { IAppDeploys, IAppSettings } from './menubar';
 import { INetlifySite } from './netlify';
+import { shortenString } from './util';
 
 interface IDeployMenuOptions {
   currentSite: INetlifySite;
   deploys: IAppDeploys;
   onItemClick: (deployId: string) => void;
 }
 
+export const getIncidentsMenu = (
+  incidentFeed: IncidentFeed
+): MenuItemConstructorOptions[] => {
+  const recentIncidents = incidentFeed
+    .getFeed()
+    // remove incidents in distant past
+    .filter(incident => {
```

Comment:
Nit: You could put this function into a variable outside of the menu and then we could spare the comment. :) 

```
const isNotOlderThanAMonth = (incident) => { /* all the logic */ }

const recentIncidents = incidentFeed
  .getFeed()
  .filter(isNotOlderThanAMonth)
  ...
```